merlin/spec/defaults.py

```diff
@@ -52,3 +52,10 @@
     "generate": {"cmd": "echo 'Insert sample-generating command here'"},
     "level_max_dirs": 25,
 }
+
+VLAUNCHER_VARS = {
+    "MERLIN_NODES": 1,
```

Comment:
Are these overridden by the step values?  WE should let the step values have precedence.